node/tests/common/test_peer.rs

```diff
@@ -113,7 +113,6 @@ impl TestPeer {
     }
 }
 
-#[async_trait::async_trait]
```

Comment:
didn't we say to keep this for now?

Reply:
Yeah, I haven't touched it in the rest of the codebase but I did update pea2pea which no longer uses these traits.